dallinger/docker/tools.py

```diff
@@ -84,7 +84,7 @@ def copy_docker_compse_files(self):
                 )
             )
         with open(os.path.join(self.tmp_dir, ".env"), "w") as fh:
-            fh.write(f"COMPOSE_PROJECT_NAME=${self.experiment_name}\n")
```

Comment:
@pmcharrison The ampersand had to be be removed in order to fix the `dockertests`.